index.html

```diff
@@ -74,6 +74,7 @@
           <h2>Heading</h2>
           <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
           <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+          <!-- Fixed Image Gallery -->
```

Comment:
Consider adding a toggle state to the button for adding images. Also adding an aria value to assist with identifying the toggle button behavior.

Reply:
Agreed with @bvasilop's suggestion. Adding the `aria-pressed` will additionally meet accessibility criteria, making it easier for users with visual impairments to know what they're interacting with.